templates/accept_general_invitation.html

```diff
@@ -0,0 +1,51 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Invitation</title>
+    <style>
+        .container {
+            padding: 20px;
+        }
+        .divider {
+            width: 58%;
+            border-top: 1px solid #ccc;
+        }
+        .button {
+          padding: 10px 20px;
+          font-weight: bold;
+          background-color: #007bff;
+          color: #fff;
+          text-decoration: none;
+            border-radius: 5px;
+        }
+        .iconify {
+            width: 7rem;
+            height: 7rem;
+            color: rgba(0, 123, 255, 0.92);
+        }
+        p{
+            font-family: sans-serif;
+        }
+    </style>
+</head>
+<body>
+    <div class="container">
+        <div>
+            <span class="iconify" data-icon="line-md:email"></span>
+        </div>
+        <h1>Hello!</h1>
+        <div class="divider">
+    </div>
+    <div>
+        <p>Congratulations! You're one step away from joining fairhub.io and discovering a whole new experience tailored just for you.</p>
+        <p>Please click the button to signup.</p>
+        <a href="{{ accept_url }}" class="button">Signup</a>
```

Comment:
The `accept_url` should also be provided as text as well. Maybe another line underneath so users can copy and paste